boards/arm/da1469x_dk_pro/da1469x_dk_pro.dts

```diff
@@ -79,7 +80,7 @@
 
 		boot_partition: partition@2400 {
 			label = "mcuboot";
-			reg = <0x000002400 0x00005c00>;
+			reg = <0x000002400 0x00020000>;
```

Comment:
How can the size be 0x20000 when the next image starts at 0x8000?

Reply:
In addition to this, unless the scratch partition is really needed because you're transferring between 2 areas with different sector sizes, remove it and increase the size of both slots